Module_6_online_shop/urls.py

```diff
@@ -15,8 +15,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('catalog/', include('catalog.urls')),
```

Comment:
Зарегистрируй этот эндпоинт в корне, чтобы при открытии сайта можно было попасть на главную страницу без дополнительных переходов
```suggestion
    path('', include('catalog.urls')),
```